cli/cmd/completion.go

```diff
@@ -0,0 +1,290 @@
+// Copyright 2020 The Lokomotive Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/spf13/cobra"
+)
+
+const completionDesc = `
+Generate autocompletion script for lokoctl for the specified shell (bash or zsh).
+`
+
+const completionExample = `# Installing bash completion on macOS using homebrew
```

Comment:
Looking at current help message format:
```
Usage:
  lokoctl completion [command]

Examples:
# Installing bash completion on macOS using homebrew
## If running Bash 3.2 included with macOS
  brew install bash-completion
## or, if running Bash 4.1+
  brew install bash-completion@2
## You need to add the completion to your completion directory
  lokoctl completion bash > /usr/local/etc/bash_completion.d/lokoctl && exec $SHELL

# Installing bash completion on Linux
## If bash-completion is not installed on Linux, please install the 'bash-completion' package
## via your distribution's package manager.
## You need to add the completion to your completion directory
  lokoctl completion bash > /etc/bash_completion.d/lokoctl && exec $SHELL

# Load the lokoctl completion code for bash into the current shell
  source <(lokoctl completion bash)
# Load the lokoctl completion code for zsh into the current shell
  source <(lokoctl completion zsh)

# Set the lokoctl completion code for zsh to autoload on startup
  lokoctl completion zsh > "${fpath[1]}/_lokoctl" && exec $SHELL

Available Commands:
  bash        Generate autocompletion script for bash
  zsh         Generate autocompletion script for zsh

Flags:
  -h, --help   help for completion
```

It seems example text could be indented with 2 spaces and should not use `#` characters. Can you re-format it @knrt10 to some better format?

Reply:
+1. Also, we should pay more attention to the following in our codebase in general and especially in text the users see:

- Punctuation.
  - Some lines in the examples text end in periods whereas others don't.
- Capitalization.
  - We use both `Bash` and `bash`. We should use `Bash`, and anyway we should be consistent.
  - Some lines begin with a capital letter whereas others don't.
- Consistent tone and style.
  - We should be consistent about using the imperative tone and avoiding "please" as in "please do this and that".
  - In one instance we use `To load completions in your current shell session:` and then immediately following we use `To load completions for every new session, execute once:`.
- Correct English.
  - I'm fairly certain the following sentence has some words missing in it (probably `that's` before `included`): `If running Bash 3.2 included with macOS`.